dts/arm/ti/am62x_m4.dtsi

```diff
@@ -48,6 +48,12 @@
 		#mbox-cells = <1>;
 	};
 
+	padcfg: padcfg@4080000 {
+		compatible = "syscon";
+		reg = <0x04080000 DT_SIZE_K(32)>;
+		status = "okay";
```

Comment:
Status is default "okay", this line then isn't needed (unless you are updating an existing disabled node which is not the  case here).

Reply:
syscon uses the DT_INST_FOREACH_STATUS_OKAY macro to initialize all compatible nodes with status set as "okay"